src/store/getters/state-getters.ts

```diff
@@ -551,3 +551,14 @@ export const showFeeSummary = (state: StateIF): boolean => {
   )
   return (haveFilingChange && !isEqual(getFilingData(state), defaultFilingData))
 }
+
+/** Return a boolean indicating if the minimum share classes requirements are met. */
+export const invalidMinimumShareClass = (state: StateIF): boolean => {
```

Comment:
Sorry to be picky but this doesn't return anything. It's a getter, so it "is" a particular value.

Also, the comment is the opposite of what it is 😛

Reply:
I don't follow? All the getters are a type of value? Whether it is returning a value or `is` a particular value. It still has a type no?

For sure, will fix the comment :)

Reply:
Updated 👍

Reply:
Ah.. you were referring the Comment `Returns` 😆 

Fixed!

Reply:
Yup, yup, thanks.